plugin/storage/grpc/config/config.go

```diff
@@ -16,11 +16,14 @@ package config
 
 import (
 	"fmt"
+	"github.com/opentracing/opentracing-go"
```

Comment:
please move to the next group (I think `make fmt` will do this automatically)

Reply:
Fixed